pkg/workspace/workspace.go

```diff
@@ -256,6 +277,57 @@ func getAccessTokens(cm *core.ConfigMap) ([][]userpkg.Role, error) {
 	}
 }
 
+func (w *workspaceImpl) Create(k8sclient kubernetes.Interface, creationData CreationData) (string, error) {
+
+	nsName := &apicore1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   creationData.Name,
+			Labels: creationData.Labels,
+		},
+	}
+	_, err := k8sclient.CoreV1().Namespaces().Create(context.Background(), nsName, metav1.CreateOptions{})
+	if err != nil {
+		return "", fmt.Errorf("error %s", err)
+	}
+
+	var strBuffer bytes.Buffer
+	strBuffer.WriteString("[")
+	for _, role := range creationData.Roles {
+		strBuffer.WriteString("\"")
+		strBuffer.WriteString(role)
+		strBuffer.WriteString("\", ")
+	}
+	roles := strBuffer.String()
+	roles = roles[:len(roles)-2] + "]"
```

Comment:
Just checking does this handle this use case?
https://github.com/equinor/flowify-workflows-server/blob/main/e2etest/workspace_cm_test.yaml#L28
could you add a test for it?